packages/mui-material/src/Autocomplete/Autocomplete.js

```diff
@@ -473,6 +474,10 @@ const Autocomplete = React.forwardRef(function Autocomplete(inProps, ref) {
   const hasPopupIcon = (!freeSolo || forcePopupIcon === true) && forcePopupIcon !== false;
 
   const { onMouseDown: handleInputMouseDown } = getInputProps();
+  const { ref: customlistboxRef } = ListboxProps ?? {};
```

Comment:
```suggestion
  const { ref: customListboxRef } = ListboxProps ?? {};
```

or `externalListboxRef`, as it's sometimes called